account_mail_templates/i18n/lt.po

```diff
@@ -22,15 +17,97 @@ msgstr ""
-msgid "<span name=\"span_doc_text\">Here is your invoice</span>"
-msgstr "<span name=\"span_doc_text\">Tai yra Jūsų sąskaita-faktūra</span>"
+msgid "$ 143,750.00"
+msgstr ""
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid "(with reference:"
+msgstr "(nuoroda:"
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid "--<br/>Mitchell Admin"
+msgstr ""
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid ""
+"<br/><br/>\n"
+"                    Please use the following communication for your payment:"
+msgstr ""
+"<br/><br/>\n"
+"                    Norėdami sumokėti, naudokite šią nuorodą:"
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid ""
+"<br/><br/>\n"
+"            Do not hesitate to contact us if you have any questions."
+msgstr ""
+"<br/><br/>\n"
+"            Jei kyla klausimų, nedvejodami susisiekite su mumis."
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid ""
+"<br/><br/>\n"
+"            Here is your"
+msgstr ""
+"<br/><br/>\n"
+"            Štai jūsų"
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid "Azure Interior"
+msgstr ""
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid "Brandon Freeman"
+msgstr ""
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid "Dear"
+msgstr "Gerb."
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid "INV/2021/05/0005"
+msgstr ""
 
 #. module: account_mail_templates
 #: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
 msgid "Please remit payment at your earliest convenience."
-msgstr "Prašome sumokėti artimiausiu galimu metu."
+msgstr "Prašome sumokėti artimiausiu jums patogiu laiku."
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid "SUB003"
+msgstr ""
 
 #. module: account_mail_templates
 #: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
 msgid "This invoice is already paid."
-msgstr "Ši sąskaita-faktūra jau apmokėta."
+msgstr "Ši sąskaita-faktūra jau yra apmokėta."
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid "YourCompany"
+msgstr ""
+
+#. module: account_mail_templates
+#: model_terms:ir.ui.view,arch_db:account_mail_templates.mail_body_invoice_send
+msgid "amounting in"
+msgstr "<span/>"
```

Comment:
Is this translation correct?

Reply:
Yes. If we would translate directly in LT, it would sound weird. So what I did, is make this translation empty for LT (can't use empty string, because it would use source). And then append that after sum in other term.

Reply:
Oh, ok.